src/module-1/debounce/index.js

```diff
@@ -1,5 +1,11 @@
 export const debounce = (fn, delay = 0) => {
+    let isCooldown = false;
 
+    return function() {
+      if (isCooldown) return;
+      isCooldown = true;
+      setTimeout(() => isCooldown = false, delay);
+      return fn.apply(this, arguments);
+    };
 }
 
```

Comment:
```
const f = debounce(alert, 1000);
f(1); // should be called
f(2); // should be ignored
setTimeout( () => f(3), 100); // should be ignored because just 100 ms passed
```

Please check your function one more time, third alert shouldn't be displayed